icicle-compiler/src/Icicle/Sea/FromAvalanche/State.hs

```diff
@@ -102,6 +104,7 @@ stateOfPrograms name attrib programs@(program :| _)
         , stateInputType      = factType
         , stateInputVars      = fmap (first textOfName) factVars
         , stateComputes       = NonEmpty.zipWith (stateOfProgramCompute name) (0 :| [1..]) programs
+        , stateOutputsAll     = Map.fromList . concatMap outputsOfProgram $ NonEmpty.toList programs
```

Comment:
I'm not sure if maybe I should do this below in `Psv/Output.hs` instead of putting something else in the data structure